plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerConfig.java

```diff
@@ -60,4 +61,16 @@ public SqlServerConfig setSnapshotIsolationDisabled(boolean snapshotIsolationDis
         this.snapshotIsolationDisabled = snapshotIsolationDisabled;
         return this;
     }
+
+    public boolean isEnableStringPushdownWithCollate()
+    {
+        return enableStringPushdownWithCollate;
+    }
+
+    @Config("sqlserver.experimental.enable-string-pushdown-with-collate")
```

Comment:
why do you stress collate? 

`sqlserver.varchar-predicate-pushdown.enabled`?

Why it should be disabled ever? Maybe we could remove this toggle all together?

Reply:
So then we should not document this and make it enabled by default and potentially in the future we should just remove it then. WDYT?

Reply:
So plan is enabled by default -> give to customer -> if any regression he could turn off it -> if everything ok, remove toggle and make this behaviour default